### PR TITLE
Update flow and wc versions

### LINKED ISSUE #7: Migrate to Flow 3.0
Like all other components mentioned in https://github.com/vaadin/flow/issues/7338, DateTimePicker should be also migrated to Flow 3.0 at some point so it can be used in V15+. 
Please note that its SNAPSHOT build for V15 in [Bender](https://bender.vaadin.com/viewType.html?buildTypeId=Flow_Components_ComponentSnapshotsFlow30v15_DateTimePicker) is paused at the moment. It should be activated when this issue is closed.

### PR DESCRIPTION
**Pull request by pekam**
close #7

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -7,20 +7,20 @@
...
         <groupId>com.vaadin</groupId>
         <artifactId>flow-component-base</artifactId>
-        <version>2.0-SNAPSHOT</version>
+        <version>3.1-SNAPSHOT</version>
     </parent>
 
     <properties>
-        <timepicker.version>2.0-SNAPSHOT</timepicker.version>
-        <datepicker.version>2.0-SNAPSHOT</datepicker.version>
+        <timepicker.version>3.0-SNAPSHOT</timepicker.version>
```
Comment (alvarezguille) [inline pom.xml:14]:
```suggestion
        <timepicker.version>3.1-SNAPSHOT</timepicker.version>
```

File: pom.xml [hunk 2]
```diff
@@ -7,20 +7,20 @@
...
+        <datepicker.version>3.0-SNAPSHOT</datepicker.version>
```
Comment (alvarezguille) [inline pom.xml:15]:
```suggestion
        <datepicker.version>3.1-SNAPSHOT</datepicker.version>
```

File: vaadin-date-time-picker-flow-integration-tests/pom.xml [hunk 3]
```diff
@@ -49,7 +49,7 @@
         <dependency>
             <groupId>com.vaadin</groupId>
             <artifactId>vaadin-grid-flow</artifactId>
-            <version>4.0-SNAPSHOT</version>
+            <version>5.0-SNAPSHOT</version>
```
Comment (alvarezguille) [inline vaadin-date-time-picker-flow-integration-tests/pom.xml:52]:
```suggestion
            <version>5.1-SNAPSHOT</version>
```

File: vaadin-date-time-picker-flow-demo/pom.xml [hunk 4]
```diff
@@ -55,7 +55,7 @@
         <dependency>
             <groupId>com.vaadin</groupId>
             <artifactId>vaadin-ordered-layout-flow</artifactId>
-            <version>2.0-SNAPSHOT</version>
+            <version>3.0-SNAPSHOT</version>
```
Comment (alvarezguille) [inline vaadin-date-time-picker-flow-demo/pom.xml:58]:
```suggestion
            <version>3.1-SNAPSHOT</version>
```

File: pom.xml (hunk 1 above)
Comment (pekam) [reply pom.xml]: Thanks!

File: pom.xml [hunk 5]
```diff
@@ -7,20 +7,20 @@
...
     </properties>
 
     <artifactId>vaadin-date-time-picker-flow-parent</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>2.0-SNAPSHOT</version>
     <packaging>pom</packaging>
     <name>Vaadin DateTimePicker Flow Parent</name>
 
-   <repositories>
+    <repositories>
```
Comment (DiegoCardoso) [inline pom.xml:23]: Extra space?

Comment (pekam) [reply pom.xml:23]: No, auto-formatting fixed it from 3 to 4 spaces.

**Pull request merged**